action.yaml

```diff
@@ -22,20 +22,28 @@ runs:
   using: "composite"
 
   steps:
+    - name: "Check for new changes in dependencies"
+      uses: brettcannon/check-for-changed-files@v1
+      with:
+        file-pattern: |
+          package.json
+          requirements.txt
+        failure-message: "No changes in ${file-pattern}"
+
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
```

Comment:
We should use the input specified by the user and not hard-code deps files. This is because, deps could be in package-lock.json or in Gemfile.lock. We should not assume this. Use file paths from `input_files`.